backend/molgenis-emx2-beacon-v2/src/main/java/org/molgenis/emx2/beaconv2/endpoints/Info.java

```diff
@@ -30,10 +30,15 @@ private JsonNode getEndpointInfo(Schema schema) {
     }
     if (schema.getTable(ENDPOINT_TABLE) == null) return null;
 
-    String query = "SELECT * FROM \"" + schema.getName() + "\".\"" + ENDPOINT_TABLE + "\"";
-    List<Row> rows = database.getSchema(schema.getName()).retrieveSql(query);
+    String query =
```

Comment:
to prevent malignant injections, it would be better to use 'jooq' here. I.e. schema.getJooq() and then retrieve. The resultset can be passed in rows if you like.

Reply:
We are programming against the interface here. so jooq is not available should I cast the SqlSchema or SqlDb? Feels wrong. Also we concatenate the sql with static final strings..

Reply:
indeed, because you use 'schema.getName()' we can be sure it is not malignant